spring/boot3-actuator-autoconfigure/src/test/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfigurationSecureTest.java

```diff
@@ -45,7 +46,7 @@
 @DirtiesContext
 @AutoConfigureMetrics
 @EnableAutoConfiguration
-@ImportAutoConfiguration(ArmeriaSpringActuatorAutoConfiguration.class)
+@ImportAutoConfiguration({ ArmeriaSpringActuatorAutoConfiguration.class, JmxEndpointAutoConfiguration.class })
```

Comment:
Applying JmxEndpointAutoConfiguration, we can reproduce #4874 .
(Occurs when the Endpoint is registered in a bean.)

code hints:
- https://github.com/be-hase/spring-boot/blob/c4de86c244acdcff69ed0aecacd254399be79ce2/spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/annotation/EndpointDiscoverer.java#L130-L139
-  https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/condition/OnAvailableEndpointCondition.java